generators/bootstrap-application-server/generator.mts

```diff
@@ -22,14 +22,15 @@ import { GENERATOR_BOOTSTRAP_APPLICATION_BASE } from '../generator-list.mjs';
 import constants from '../generator-constants.cjs';
 import entityUtils from '../../utils/entity.cjs';
 import type { SpringBootApplication } from './types.js';
+import fieldTypes from '../../jdl/jhipster/field-types.js';
+import authenticationTypes from '../../jdl/jhipster/authentication-types.js';
 import { prepareFieldForLiquibaseTemplates } from '../../utils/liquibase.cjs';
-import AuthentitcationTypes from '../../jdl/jhipster/authentication-types.js';
-import FieldTypes from '../../jdl/jhipster/field-types.js';
 
-const {
-  CommonDBTypes: { LONG: TYPE_LONG },
-} = FieldTypes;
-const { OAUTH2 } = AuthentitcationTypes;
+const { CommonDBTypes } = fieldTypes;
+const { OAUTH2 } = authenticationTypes;
+
+const { LONG: TYPE_LONG } = CommonDBTypes;
```

Comment:
There should be an issue to improve the imports.

Reply:
The imports are rather ugly in part because we use multiple module systems. This will improve when the project is completely migrated to ESM. See this issue: #19113